src/components/ProductPopover.vue

```diff
@@ -5,11 +5,11 @@
     </ion-item>
     <ion-item lines="none" @click="revert">
       <ion-icon slot="start" :icon="arrowUndoOutline" />
-      <ion-label>{{ $t('Reset') }}</ion-label>
+      <ion-label class="pointer">{{ $t('Reset') }}</ion-label>
```

Comment:
We can add button to `ion-item` and then we don't need to use class.

Reply:
https://ionicframework.com/docs/api/item#clickable-items